getting_started/step_by_step/your_first_game.rst

```diff
@@ -709,24 +709,24 @@ function to set everything up for a new game:
         //timers
         var mobTimer = (Timer) GetNode("MobTimer");
         var scoreTimer = (Timer) GetNode("ScoreTimer");
-    
+
         scoreTimer.Stop();
         mobTimer.Stop();
     }
 
     public void NewGame()
     {
         Score = 0;
-    
+
         var player = (Player) GetNode("Player");
         var startTimer = (Timer) GetNode("StartTimer");
         var startPosition = (Position2D) GetNode("StartPosition");
-        
+
         player.Start(startPosition.Position);
         startTimer.Start();
     }
 
-Now connect the ``timeout()`` signal of each of the Timer nodes.
+Now connect the ``timeout()`` signal of each of the Timer nodes (``StartTimer``, ``ScoreTimer``,and ``MobTimer``).
```

Comment:
You missed the space after the comma. Then can you please squash the commits so there's only one? It helps keep the commit log clean.